pulsar-client/src/main/java/com/yahoo/pulsar/client/util/Version.java

```diff
@@ -0,0 +1,52 @@
+/**
+ * Copyright 2016 Yahoo Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.yahoo.pulsar.client.util;
+
+import java.io.FileReader;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.maven.model.Model;
+import org.apache.maven.model.io.xpp3.MavenXpp3Reader;
+
+public class Version {
+
+    private static String VERSION = null;
+
+    /**
+     * Fetch version of the artifact which could be stored at MANIFEST.MF in jar file or pom of the stand-alone project
+     * 
+     * @return
+     */
+    public static String getVersion() {
```

Comment:
The client JAR already includes a `pulsar-client-version.properties` file that contains the client version. That file is generated at build time with the info from pom. 

Take a look at `HttpClient.getPulsarClientVersion()` for how to get the info